buildbuddy.yaml

```diff
@@ -62,5 +62,6 @@ plugins:
   - path: cli/plugins/go-deps
   - path: cli/plugins/open-invocation
   - path: cli/plugins/notify
+  - path: cli/plugins/config-per-target
```

Comment:
Sorry for the drive-by, but the target level `args` parameter just be used for this? For example:
https://github.com/buildbuddy-io/buildbuddy/blob/master/server/cmd/buildbuddy/BUILD#L9

I'm not sure it's worth complicating the plugin API for this, i.e. making plugins take two arguments parameters instead of one.

Reply:
The functionality was supposed to be a natural extension of the current plugin `pre-bazel` functionality; I, at least, was (very) surprised that the plugins could not modify the "exec args", basically, and I wanted to be able to write a plugin that can. I would like to be able to create configurations on a per-target basis that do not need to be written to BUILD files and thus do not need to be included in the git repository. I was using `tools/bazel` to do this, but it was comparatively brittle.